src/docs/style/iconography.mdx

```diff
@@ -9,6 +9,7 @@ specific to PX Blue products.
 
 These icons are available in a variety of formats - select an icon below to view its usage instructions.
 
+<span /><MaterialDesignDescription title={'System icons'} url={'https://material.io/design/iconography/system-icons.html#'} /><MaterialDesignDescription title={'Material\'s icon'} url={'https://material.io/resources/icons/'} description={'List of Material Design icons. They are also included in the table below.'} style={{marginRight:0}}/>
```

Comment:
- What's this `<span>` doing here?

Reply:
That was from a weird behavior my prettier is doing.

Reply:
that was because there is something strange going on with the Prettier. It keeps merging line 10 and line 12, resulting in the cards being put in the `<p>` tag.

Reply:
very very strange. I see that too, but only when I run prettier multiple times in a row. It really doesn't like having that third component in a row being a self-closing tag for some reason. If you wrap the three components in a `<React.Fragment>...</React.Fragment>` it seems to keep prettier at bay.